volume/drivers/fake/fake_test.go

```diff
@@ -254,6 +260,11 @@ func TestFakeCloudBackupEnumerateWithoutMatches(t *testing.T) {
 	})
 	assert.NoError(t, err)
 	assert.Len(t, resp.Backups, numbackups)
+	for _, id := range taskIds {
+		_, err := d.kv.Delete(backupsKeyPrefix + "/" + id)
```

Comment:
Call CloudBackupDelete() instead of deleting from kvdb directly?

Reply:
This is not deleting cloudsnaps, rather cleaning up kvdb that is  used by fake driver.